### PR TITLE
Wishlist and cart Completed

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: src/Components/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+.product-remove:hover {
+	color: var(--bg-color-1);
+	color: var(--primary-color-dark);
+	/* font-weight: 600; */
+}
+
+.adjustment-container {
+	position: absolute;
+	bottom: 70px;
```
Comment (MayankKumar10) [inline src/Components/Cart/cart.css:78]: use rem instead of px

File: src/Components/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	background-color: var(--primary-dark);
+}
+
+@media screen and (max-width: 700px) {
+	.remove {
+		display: none;
+	}
+
+	.product {
+		height: 150px;
```
Comment (MayankKumar10) [inline src/Components/Cart/cart.css:139]: use rem instead of px

**Review by MayankKumar10 — APPROVED** [review]
Look into these changes everything else looks good to me you can merge it.

**Review by sainath7878 — APPROVED** [review]
Code looks neat and clean. However, I have few suggestions in UI and functionality

- Contrast Ratio is too low for the badges in product Card and proceed to checkout text in cart page.
- Quantity cannot be negative in cart page.
- Add to cart and Add to Wishlist is not working with test login.

Things I liked:

- Nice folder structure.
- Different contexts for cart, wishList and Auth.

**Pull request merged**